src/connector/src/parser/debezium/avro_parser.rs

```diff
@@ -108,18 +111,16 @@ impl DebeziumAvroParserConfig {
         let kafka_topic = &avro_config.topic;
         let url = handle_sr_list(schema_location)?;
         let client = Client::new(url, client_config)?;
-        let raw_schema = client
-            .get_schema_by_subject(format!("{}-key", &kafka_topic).as_str())
-            .await?;
-        let key_schema = Schema::parse_str(&raw_schema.content)
-            .map_err(|e| RwError::from(ProtocolError(format!("Avro schema parse error {}", e))))?;
-
         let resolver = ConfluentSchemaResolver::new(client);
-        let outer_schema = resolver
-            .get_by_subject_name(&format!("{}-value", kafka_topic))
-            .await?;
```

Comment:
@tabVersion Any idea why `key_schema` skips the resolver cache in #7968?

Reply:
I think it is an unintentional misuse